staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -15,9 +15,24 @@
 package model
 
 import (
+	"time"
+
+	"github.com/google/cel-go/checker/decls"
+	"github.com/google/cel-go/common/types"
 	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
 )
 
+const (
+	maxRequestSizeBytes = 3000000
```

Comment:
comment on this for sure. Also reference where in the code the 3MB limit is defined if possible (and if we can reference the const value here instead of re-declaring it, that would be amazing)

Reply:
Added a comment - I'm marking this as resolved since the TODO ought to be fixed before merging.